projects/backend/controllers/fronts.ts

```diff
@@ -13,7 +13,7 @@ export const frontController = (req: Request, res: Response) => {
         req.params.version,
         isPreview,
     )
-    const edition = req.params.edition
+    const edition = getEditionOrFallback(req.params.edition)
```

Comment:
If this is just a `yarn upgrade` then why do we have changes to logic?

Reply:
I was wondering that! Basically after I ran yarn upgrade the `validate` step started failing - perhaps because of babel changes? Doesn't really make sense though.

Reply:
You can see the failure stuff [here](https://github.com/guardian/editions/runs/890101085) - basically it was angry about treating strings as typed values